plugins/signalmonitor/signalhistorymodel.cpp

```diff
@@ -182,6 +202,31 @@ QMap< int, QVariant > SignalHistoryModel::itemData(const QModelIndex &index) con
     return d;
 }
 
+bool SignalHistoryModel::setData(const QModelIndex &index, const QVariant &value, int role)
+{
+    switch (static_cast<ColumnId>(index.column())) {
```

Comment:
considering only one column is editable, why not use an if here instead of a switch?